eng/ProjectReferences.props

```diff
@@ -16,6 +16,7 @@
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.SignalR.Specification.Tests" ProjectPath="$(RepoRoot)src\SignalR\server\Specification.Tests\src\Microsoft.AspNetCore.SignalR.Specification.Tests.csproj"  />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Blazor.Build" ProjectPath="$(RepoRoot)src\Components\Blazor\Build\src\Microsoft.AspNetCore.Blazor.Build.csproj"  />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Blazor.DataAnnotations.Validation" ProjectPath="$(RepoRoot)src\Components\Blazor\Validation\src\Microsoft.AspNetCore.Blazor.DataAnnotations.Validation.csproj"  />
+    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Blazor.DevServer" ProjectPath="$(RepoRoot)src\Components\Blazor\DevServer\src\Microsoft.AspNetCore.Blazor.DevServer.csproj" />
```

Comment:
This looks **Just Wrong:tm:** because the named project has `<IsProjectReferenceProvider>false</IsProjectReferenceProvider>`

Reply:
Should we remove the `IsProjectReferenceProvider` from the project? I imagine we want to reference this from the test.

Reply:
Easier solution: I remove this addition which was only there for when I tested something locally anyway.

Reply:
I'm assuming you didn't push that commit yet

Reply:
Huh, somehow I missed both the fact that my removal got reverted at some point and your comment pointing that out. Should be fixed now.